app/code/Transoft/Blog/Model/Api/SearchCriteria/CollectionProcessor/FilterProcessor/ModelStoreFilter.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Why do we need this. You **don't** have **store_id** field in blog table

Reply:
removed